test/integration/etcd_storage_path_test.go

```diff
@@ -1050,6 +1022,12 @@ func testEtcdStoragePath(t *testing.T, etcdServer *etcdtest.EtcdTestServer, gett
 		t.Fatalf("error creating test namespace: %#v", err)
 	}
 
+	etcdClient3, err := etcd.MakeEtcdClientV3(masterConfig.EtcdClientInfo)
+	if err != nil {
+		t.Fatal(err)
+	}
+	getter := &etcd3Getter{kv: etcdClient3.KV}
```

Comment:
The etcd3Getter struct seems a bit pointless now since you no longer have the interface.